staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/customresource_handler.go

```diff
@@ -546,14 +556,30 @@ func (c unstructuredCreator) New(kind schema.GroupVersionKind) (runtime.Object,
 }
 
 type unstructuredDefaulter struct {
-	delegate runtime.ObjectDefaulter
+	delegate  runtime.ObjectDefaulter
+	defaulter *validate.SchemaValidator
 }
 
 func (d unstructuredDefaulter) Default(in runtime.Object) {
 	// Delegate for things other than Unstructured.
-	if _, ok := in.(runtime.Unstructured); !ok {
+	obj, ok := in.(runtime.Unstructured)
+	if !ok {
 		d.delegate.Default(in)
```

Comment:
tolerate not having a delegate

Reply:
I should be able to use a no-op unstructuredDefaulter (is what i meant by toleration)

Reply:
true. it's old code, but will fix anyway.